scripts/e2e_test.sh

```diff
@@ -41,7 +41,7 @@ $FORGE_COMMAND
 cd $cwd
 # Build ginkgo
 # to install the ginkgo binary (required for test build and run)
-go install -v github.com/onsi/ginkgo/v2/ginkgo@${GINKGO_VERSION}
+go install -v github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
What was the reason behind this change?

Reply:
Oh interesting... `GINKGO_VERSION` is supposed to come from `go.mod`, but I guess in this repo it's pulling the version from the `teleporter` submodule, which we don't want to have to keep in sync, so this change makes sense :)